csharp/ql/src/semmle/code/csharp/Generics.qll

```diff
@@ -350,15 +350,18 @@ class UnboundGenericDelegateType extends DelegateType, UnboundGenericType {
 }
 
 /**
- * A constructed (bound) type. This is a generic type for which actual type arguments have been supplied,
- * for example `G<int>` or the `G<T>` in `class Other<T> { G<T> g; }`. Constructed types can be divided further into
- * those that are open (for example `G1<T>` or `G2<T,T,U,int>`), in the sense that one or more of their type arguments
- * is a type parameter, versus those that are closed (for example `G1<int>` or `G2<long,long,float,int>`). We do not
- * currently distinguish the two in this library.
- *
- * Either a constructed `struct` (`ConstructedStruct`), constructed `class` (`ConstructedClass`),
- * constructed `interface` (`ConstructedInterface`), or constructed method
- * (`ConstructedMethod`).
+ * A constructed (bound) type. This is a generic type for which actual type
+ * arguments have been supplied, for example `G<int>` or the `G<T>` in
+ * `class Other<T> { G<T> g; }`. Constructed types can be divided further into
+ * those that are open (for example `G1<T>` or `G2<T,T,U,int>`), in the sense
+ * that one or more of their type argumentsis a type parameter, versus those
+ * that are closed (for example `G1<int>` or `G2<long,long,float,int>`).
+ *
+ * We do not currently distinguish the two in this library.
```

Comment:
I don't think this line makes sense as it would be fairly easy to model this in QL if it turned out to be needed.